jekyll/_cci2/testing-ios.md

```diff
@@ -98,7 +132,7 @@ job as follows:
 
 
 ```yaml
-version: 2
+version: 2.1
```

Comment:
I believe if we change this to 2.1, the `version 2` bit from the workflows section should be removed :-)